mittab/libs/data_import/export_xls_files.py

```diff
@@ -0,0 +1,220 @@
+from django.contrib.auth.decorators import permission_required
+from django.http import HttpResponse
+from xlwt import Workbook
+
+from mittab.apps.tab.models import Team, Judge, Room, Debater
+from mittab.libs import tab_logic
+
+
+def _create_vn_str(novice_status):
+    """Creates varsity-novice status string from the integer pseudo-enum used by the model"""
+    if novice_status is 0: return "varsity"
```

Comment:
Use `Debater.NOVICE` and `Debater.VARSITY` rather than specifically referencing 0 and 1. That way, changing values in the `Debater` model won't break this logic

Reply:
Actually, this method can be:

```python
for status, description in Debater.NOVICE_CHOICES:
    if status == novice_status: return status
return ""
```